notebooks/02_dj_webinar.ipynb

```diff
@@ -0,0 +1,1403 @@
+{
```

Comment:
<p>reminder to consider drop the word ERD in favor of relationship diagram or simply Diagram</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/datajoint-webinar/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>